src/helm/benchmark/run_specs/bhasa_run_specs.py

```diff
@@ -0,0 +1,108 @@
+from helm.benchmark.adaptation.adapter_spec import (
+    ADAPT_MULTIPLE_CHOICE_SEPARATE_ORIGINAL,
+)
+from helm.benchmark.adaptation.common_adapter_specs import (
+    get_generation_adapter_spec,
+    get_multiple_choice_separate_adapter_spec,
+)
+from helm.benchmark.metrics.common_metric_specs import (
+    get_exact_match_metric_specs,
+)
+from helm.benchmark.run_spec import RunSpec, run_spec_function
+from helm.benchmark.scenarios.scenario import ScenarioSpec
+
+# BHASA Run Specs
+#   D. Linguistic Diagnostics
+
+# D. Linguistic Diagnostics (LINDSEA)
+#   1. Syntax
+#   2. Pragmatics
+
+# 1. Syntax: LINDSEA Minimal Pairs
+LINDSEA_OUTPUT_NOUNS = {
+    "id": "Jawaban"
+}
+
+@run_spec_function("lindsea_syntax_minimal_pairs")
+def get_lindsea_syntax_minimal_pairs_spec(language: str = "id", method: str = "mcq") -> RunSpec:
+    name = f"lindsea_syntax_minimal_pairs_{language}"
+    if method == "mcq":
```

Comment:
What's the reason for doing the "A" and "B" formatting yourself, instead of using `get_multiple_choice_joint_adapter_spec()` which will do it for you? `get_multiple_choice_joint_adapter_spec()` has the added advantage of being more similar to `get_multiple_choice_separate_adapter_spec()` except for logprobs vs generation.